webapp/lib/charts.dart

```diff
@@ -145,7 +145,7 @@ class DailyTimeseriesLineChartView {
     chart = chartjs.Chart(canvas.getContext('2d'), chartConfig);
   }
 
-  void updateChart([List<Map<DateTime, num>> updatedCountsAtTimestampList, String timeScaleUnit = 'day']) {
+  void updateChart(List<Map<DateTime, num>> updatedCountsAtTimestampList, {String timeScaleUnit = 'day', num upperLimit = null}) {
```

Comment:
I don't think you need to explicitly set `upperLimit` to `null` - Dart will do that by default and it's probably best to keep the explicit assignments to places where we set them to a non-null value.

Reply:
Okay. Fixed and merging